### ISSUE TITLE
MMB freezes when GPM6_IMF13 comparison is made with all possible rules

### ISSUE THREAD

**Issue by btatar13** [IMFS-MMB/mmb-gui-electron#102]
**Describe the bug**
GPM6_IMF13 crashes if comparison is made with all rules as it seems that it is too much for Matlab. Model is huge. Actually, I don't think so that we can do much about it, or can we?

**To Reproduce**
1. Compare GPM6_IMF13 with all possible rules

**Desktop (please complete the following information):**
 - Win10
 - Matlab 2019a
 - Dynare 4.5.7

**Comment by gboehl** [comment]
Nope. One could do some automated crash handling, but I really don't think we should give that any priority.

**Comment by j2L4e** [comment]
I don't think it crashes, it just takes a REALLY long time because it has so many variables. JSON serialization is quite slow but I think it's the only relatively-trivial option we have. I'd expect saving a `.mat` file and parsing that on the electron side to be orders of magnitude faster for large datasets but it doesn't seem worth the effort
